service/README.md

```diff
@@ -53,17 +53,87 @@ Change the variable for CCD_DEF_CASE_SERVICE_BASE_URL in bin/configurer/utils/fp
 the service.
 
 ### Run the application (from IntelliiJ)
- 
-Ensure the Spring Boot is started with local profile 
-(add environment variable spring.profiles.active=local when starting the main class).
 
-Configure the notify.api_key if necessary (you can pass them as environment variables when IntelliJ 
-starts the application). 
+Ensure the Spring Boot is started with local, user-mappings and the feature-toggle profiles
+(add environment variable spring.profiles.active=feature-toggle,local,user-mappings when starting the main class).
 
-Ensure the variable CCD_DEF_CASE_SERVICE_BASE_URL bin/configurer/utils/fpl-process-definition.sh is set to
-http://docker.for.mac.localhost:4000 - this will use the host machine for CCD, reaching 
-locally running application. 
+Configure the notify.api_key if necessary (you can pass them as environment variables when IntelliJ
+starts the application).
 
+Ensure the variable CCD_DEF_CASE_SERVICE_BASE_URL bin/configurer/utils/fpl-process-definition.sh is set to
+http://docker.for.mac.localhost:4000 - this will use the host machine for CCD, reaching
+locally running application.
+
+## Config
+
+Services uses three sources of configuration for the placeholders:
+* application.yaml (profile-driven, test and production systems use only default profile)
+* ENV variables (passed in via charts in the cluster or docker-compose for local)
+* Secrets from Azure Key vaults and following [library](https://github.com/hmcts/properties-volume-spring-boot-starter).
```

Comment:
Are you trying to say
```suggestion
* Secrets from Azure Key vaults and the following [library](https://github.com/hmcts/properties-volume-spring-boot-starter).
```

Or this
```suggestion
* Secrets from Azure Key vaults following the templates put forward by this [library](https://github.com/hmcts/properties-volume-spring-boot-starter).
```
maybe patterns is a better word than templates

Reply:
The whole point changed to:
* Following [library](https://github.com/hmcts/properties-volume-spring-boot-starter) provides secrets as configuration placeholders.
  [This file](src/main/resources/bootstrap.yaml) configures mapping from a secret name to Spring's configuration property.